src/utils/common-expression-helpers.ts

```diff
@@ -444,6 +444,17 @@ export class CommonExpressionHelpers {
     return Promise.resolve(lazy);
   };
 }
+export class HistoricalDataSourceService {
```

Comment:
Can we move this to its own file? I also feel like it's valid to move the expression runner components out of the utils to a new folder "expression-runner".